editor/src/com/talosvfx/talos/editor/addons/scene/widgets/ProjectExplorerWidget.java

```diff
@@ -217,21 +219,21 @@ public void deletePath(Array<String> paths) {
             @Override
             public void run () {
 
-                logger.info("todo - Reimplement delete paths");
-//                FileHandle parent = SceneEditorAddon.get().workspace.getProjectFolder();
-//                for(String path: paths) {
-//                    FileHandle handle = Gdx.files.absolute(path);
-//                    AssetImporter.deleteFile(handle);
-//
-//                    parent = handle.parent();
-//                }
-//
-//                loadDirectoryTree((String) rootNode.getObject());
-//
-//                if(!parent.path().equals(parent)) {
-//                    expand(parent.path());
-//                    select(parent.path());
-//                }
+                FileHandle parent = SharedResources.currentProject.rootProjectDir();
```

Comment:
This seems like a hack to just get the current directory

Reply:
Fixed!